integrations/gitlab_v2/client.py

```diff
@@ -0,0 +1,296 @@
+import asyncio
+from typing import Any, AsyncGenerator
+
+import httpx
+from httpx import Timeout
+from loguru import logger
+
+from rate_limiter import GitLabRateLimiter
+from port_ocean.context.ocean import ocean
+from port_ocean.utils import http_async_client
+from port_ocean.utils.cache import cache_iterator_result
+
+REQUEST_TIMEOUT: int = 60
+CREATE_UPDATE_WEBHOOK_EVENTS: list[str] = [
+    "open",
+    "reopen",
+    "update",
+    "approved",
+    "unapproved",
+    "approval",
+    "unapproval",
+]
+DELETE_WEBHOOK_EVENTS: list[str] = ["close", "merge"]
+WEBHOOK_EVENTS_TO_TRACK: dict[str, bool] = {
+    "issues_events": True,
+    "merge_requests_events": True,
+}
```

Comment:
gitlab has more webhook events. please track all of them instead of these 2

Reply:
Okay noted